src/modules/update.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that you remove the eslint disables you have on each js file and try to fix the errors manually. This will help you write better code as the eslint helps check for code standards.